pyuvdata/cst_beam.py

```diff
@@ -201,7 +209,7 @@ def read_cst_beam(self, filename, beam_type='power', feed_pol='x',
             rot_theta = theta_data
 
         # get beam
-        if self.beam_type is 'power':
+        if self.beam_type == 'power':
             data_col = np.where(np.array(column_names) == 'abs(v)')[0][0]
```

Comment:
So this was one of the lines that got tripped up using Nic's new CST `.txt` files. In this new files, the appropriate column is labeled `E-Field`, rather than `abs(v)`. Nic confirmed with me that in the new files this is indeed the appropriate column to use, but it means that our way of finding this column here will fail, which is why in the `HERA-Beams` PR I added a function for actually editing the CST files to replace `E-Field` with `V` so that the correct data column is caught.

Not sure what the best way to handle this is. My short-term solution was to include some utility functions in `convert_to_uvbeam.py` that actually re-writes the CST files with headers compliant with this standard.

Reply:
Thanks for this, Nick. I haven't really tested against the new files, but I will do that now and include needed changes in this PR. I was thinking about that as a separate step, but I agree those changes make sense in this PR.

Note that the new settings files actually list the filenames and the frequencies (in two ordered lists) so we no longer need to derive the frequency from the filename.

Reply:
@nkern I'm not seeing the problem you report with the column name. Which specific directory were you reading from?